### DISCUSSION TITLE
New versioning system

### DISCUSSION THREAD

**Discussion by puelp** [celzero/rethink-app#561] [Ideas]
053i and 053j make them both seem like minor bug-fix releases of similar importance.

But from what I see, 053i was a pretty big upgrade, so I feel like it could have easily been called 06, or 0.6.

And 053j looks like it was a bug-fix release, so I feel like it could have been called 0.6.1.

**Comment by ignoramous** [comment]
Don't read too much into our versioning. It doesn't follow semver.

What's happened with v053i is that it is partially (say, 40%) a v054 release (but not quite 100%). We decided to release the partial release because it had been 10 months since the previous release.
